Automattic/jetpack
# State: Move products state tree outside of the site tree

This PR moves the products Redux state outside of the site tree, because it doesn't really have anything to do with sites, so it should be separate.

Fixes #14038

#### Changes proposed in this Pull Request:
* State: Move products state tree outside of the site tree

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of the Jetpack Backup project, see #14038 for context.

#### Testing instructions:
* Checkout this branch.
* Build Jetpack (`yarn build` or `yarn watch`)
* Go to /wp-admin/admin.php?page=jetpack#/plans on a site with a free plan and no Jetpack Backup product purchased.
* Verify prices and product info appears properly.

#### Proposed changelog entry for your changes:
* State: Move products state tree outside of the site tree
